### ISSUE TITLE
Add option to pass custom config into integration tests

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1436]
Right now one is able to modify vanilla Minishift used in integration test with environmental variables. For example more memory can be added... However some flags used with `minishift start` can't be implemented as environmental variables and thus to support these for integration tests we need to implement a possibility to pass them as an argument to `make integration` and later apply this argument as config setup in beforeFeature().

**Comment by agajdosi** [comment]
Ok, so I have tried to do this properly with flag accepting stringArray values, which due to the 3 level nesting: makefile, go test command, flag parser for Godog tests resulted in need for quite ugly commands:

```
make integration GODOG_OPTS=-tags=nothing MINISHIFT_BINARY=/home/agajdosi/minishift/minishift CUSTOM_TEST_FLAGS="--pre-start-commands 'config set extra-clusterup-flags -- \"--image=myserver.com:8888/openshift --version=latest\"' --pre-start-commands 'config set insecure-registry \"myserver.com:8888\"'"
```

Quoting would be just mad in this case.

So I went for solution which is less elegant in field of code and elasticity, but will result in much more human friendly parameters for `make integration`. Example is bellow:

```
make integration GODOG_OPTS=-tags=future MINISHIFT_BINARY=/home/agajdosi/minishift/minishift EXEC_BEFORE_FEATURE="config set insecure-registry 'myserver.com:8888';config set extra-clusterup-flags -- '--image=myserver.com:8888/openshift --version=latest'"
```

For sure this second solution will have problems with any command which has semicolon in it, but I am not aware of any we would need right now.

**Comment by agajdosi** [comment]
I have added a prototype, if you will have time on Monday, let me know what do you think, @praveenkumar. I am able to provision newest OCP image with these changes, so probably it needs some kind of ack if it looks like reasonable and friendly way to go, I can then polish and create WIP.

**Comment by LalatenduMohanty** [comment]
Resolved by #1445
